peripherals/rcc/rcc_u5.yaml

```
@@ -0,0 +1,721 @@
+      Div120: [119, "pllx_p_ck = vcox_ck / 120"]
+      Div122: [121, "pllx_p_ck = vcox_ck / 122"]
+      Div124: [123, "pllx_p_ck = vcox_ck / 124"]
+      Div126: [125, "pllx_p_ck = vcox_ck / 126"]
+      Div128: [127, "pllx_p_ck = vcox_ck / 128"]
+    PLL?N: [3, 0x1FF]
+  PLL?FRACR:
+    PLL?FRACN: [0, 0x1FF]
+  CIER:
+    "*RDYIE":
+      Disabled: [0, "Interrupt disabled"]
+      Enabled: [1, "Interrupt enabled"]
+  CIFR:
+    "*RDYF":
+      _read:
+        NotInterrupted: [0, "No clock ready interrupt"]
+        Interrupted: [1, "Clock ready interrupt"]
+  CICR:
+    "*RDYC":
+      _write:
+        Clear: [1, "Clear flag"]
+  A[HP]B?RSTR*:
+    "*RST":
+      NoEffect: [0, "No effect"]
+      Reset: [1, "Reset peripheral"]
+  A[HP]B?ENR*:
+    "*EN":
+      Disabled: [0, "Peripheral clock disabled"]
+      Enabled: [1, "Peripheral clock enabled"]
+  A[HP]B?SMENR*:
+    "*SMEN":
+      Disabled: [0, "Peripheral clocks disabled by the clock gating during Sleep and Stop modes"]
+      Enabled: [1, "Peripheral clocks enabled by the clock gating during Sleep and Stop modes"]
+  SRDAMR:
+    "*AMEN":
+      Disabled: [0, "Peripheral autonomous mode disabled during Stop 0/1/2 mode"]
+      Enabled: [1, "Peripheral autonomous mode enabled during Stop 0/1/2 mode"]
+  CCIPR1:
+    TIMICSEL:
+      Disabled: [0, "HSI, MSIK and MSIS dividers disabled"] # same for [0, 3]
+      HsiMsisMsis: [4, "HSI/256, MSIS/1024 and MSIS/4 generated and can be selected by TIM16, TIM17, and LPTIM2 as internal input capture"]
+      HsiMsisMsik: [5, "HSI/256, MSIS/1024 and MSIK/4 generated and can be selected by TIM16, TIM17, and LPTIM2 as internal input capture"]
+      HsiMsikMsis: [6, "HSI/256, MSIK/1024 and MSIS/4 generated and can be selected by TIM16, TIM17, and LPTIM2 as internal input capture"]
+      HsiMsikMsik: [7, "HSI/256, MSIK/1024 and MSIK/4 generated and can be selected by TIM16, TIM17, and LPTIM2 as internal input capture"]
+    ICLKSEL:
+      HSI: [0, "HSI48 clock selected"]
+      PLL2Q: [1, "PLL2 \"Q\" (pll2_q_ck) selected"]
+      PLL1Q: [2, "PLL1 \"Q\" (pll1_q_ck) selected"]
+      MSIK: [3, "MSIK clock selected"]
+    FDCAN1SEL:
+      HSE: [0, "HSE clock selected"]
+      PLL1Q: [1, "PLL1 \"Q\" (pll2_q_ck) selected"]
+      PLL2P: [2, "PLL2 \"P\" (pll1_p_ck) selected"]
+    SYSTICKSEL:
+      HCLK_Div8: [0, "HCLK/8 selected"]
+      LSI: [1, "LSI selected"]
+      LSE: [2, "LSE selected"]
+    LPTIM2SEL:
+      PCLK1: [0, "PCLK1 selected"]
+      LSI: [1, "LSI selected"]
+      HSI16: [2, "HSI16 selected"]
+      MSIK: [3, "MSIK selected"]
+    SPI2SEL,I2C[124]SEL,UART[45]SEL,USART[23]SEL:
+      PCLK1: [0, "PCLK1 selected"]
+      SYSCLK: [1, "SYSCLK selected"]
+      HSI16: [2, "HSI16 selected"]
+      MSIK: [3, "MSIK selected"]
+    SPI1SEL,USART1SEL:
+      PCLK2: [0, "PCLK2 selected"]
+      SYSCLK: [1, "SYSCLK selected"]
+      HSI16: [2, "HSI16 selected"]
+      MSIK: [3, "MSIK selected"]
```

```suggestion
    SPI?SEL,I2C[124]SEL,UART[45]SEL,USART?SEL:
      PCLK: [0, "PCLKx selected"]
      SYSCLK: [1, "SYSCLK selected"]
      HSI16: [2, "HSI16 selected"]
      MSIK: [3, "MSIK selected"]
```
It is just recommendation, but I'd collect them to simplify writing a HAL.